src/Flex.php

```diff
@@ -275,7 +275,7 @@ class_exists(__NAMESPACE__.str_replace('/', '\\', substr($file, \strlen(__DIR__)
             $app->add(new Command\UnpackCommand($resolver));
             $app->add(new Command\RecipesCommand($this, $this->lock, $rfs));
             $app->add(new Command\InstallRecipesCommand($this, $this->options->get('root-dir')));
-            $app->add(new Command\GenerateIdCommand());
+            $app->add(new Command\GenerateIdCommand(null));
```

Comment:
To prepare the next step, we can maybe register the command only if it exists?

```php
if (class_exists(Command\GenerateIdCommand::class)) {
    $app->add(new Command\GenerateIdCommand(null));
}
```

Reply:
Indeed, that should clear the way to remove the command later